robot/Cumulus/resources/RecurringDonationsPageObject.py

```diff
@@ -182,6 +182,16 @@ def validate_field_values_under_section(self, section=None, **kwargs):
                     label, "contains", value, section
                 )
 
+    def get_next_payment_date_number(self, num):
+        """Returns the next payment date from the payment schedule"""
```

Comment:
can you please clarify on what is the num variable here

Reply:
Updated the documentation to be more clearer with example